go/libraries/doltcore/env/environment.go

```diff
@@ -431,8 +431,8 @@ func (dEnv *DoltEnv) configureRepo(doltDir string) error {
 
 // Inits the dolt DB of this environment with an empty commit at the time given and writes default docs to disk.
 // Writes new repo state with a main branch and current root hash.
```

Comment:
(minor) would be nice to mention the `|requiresFunHash|` param, especially since this is part of the public API for this type. We don't always do a great job with godocs for functions and with describing params, but we're trying to be more thorough there.

Reply:
`requiresFunHash` param no longer exists.